lib/sierra-models/base.js

```diff
@@ -46,7 +46,7 @@ class SierraBase {
   // This function takes a marc tag, and returns an array of primary values with
   //   relevant parallel values attached and or orphaned parallels relevant to
   //   given marc tag
-  _addParallels (marc) {
```

Comment:
This function appears to do much more than adding a parallel, so proposing name change

Reply:
👍